katdal/dataset.py

```diff
@@ -1025,9 +1039,11 @@ def sensor_data(ant_name):
 
     def _sensor_per_corrprod(self, base_name):
         """Extract a single sensor per corrprod and safely stack the results."""
```

Comment:
Maybe rename this, since the implementation is now very specific to values where the corrprod result is the difference of per-antenna results.

Reply:
I wondered about that too.

Reply:
Renamed.